config_default.php

```diff
@@ -774,6 +774,24 @@
 	*/
 	$conf['mysql_create_tbl_opts'] = 'ENGINE=InnoDB DEFAULT CHARSET=utf8';
 
+        /*
+        |===============================================
+        | Whitelist Management Console Access
+        |===============================================
+        |
+        | Whitelisting of IP addresses that can access the management interface 
+        |    (anything except for index.php?/report/ which is always allowed)
+        |  - You can provide either individual IP addresses (which will have /32 appended automatically)
+        |      or you can provide CIDR notation. See https://en.wikipedia.org/wiki/Classless_Inter-Domain_Routing for reference
+        |  - You can also provide a custom 403 page for traffic that does not have access to the management interface
+        |      Default: Value of $conf['webhost'] with 403.html added to the end
+        |
+        */
+        //$conf['manage_ipwhitelist'] = array('xxx.xxx.xxx.xxx', 'xxx.xxx.xxx.xxx/yy');
+        //$conf['403_unauth'] = 'http://fqdn/403.html';
```

Comment:
You can leave out the option for the 403 and use the built in error page?

Reply:
My organiztion would like a custom page, but how about instead of sending it to the default 403 page for the server, send it to the the built-in one? This way, we can have a custom 403 page location, and everyone else that doesnt want one will be sent to the built in error page

Reply:
Ok, how about this:
```php
$conf['auth']['network'] = [
    'whitelist_ipv4' => [
        'xxx.xxx.xxx.xxx',
        'xxx.xxx.xxx.xxx',
    ],
    'redirect_unauthorized' => 'http://fqdn/403.html',
]
```
Where `redirect_unauthorized` is optional and defaults to the built-in 403 page